### DISCUSSION TITLE
No validation ?

### DISCUSSION THREAD

**Discussion by ghost** [BetaHuhn/feedback-js#29] [Q&A] [answered]
If there is no validation for user inputs, even if it "feels" restricted by design of UI, it is very susceptible of hack.

**Comment by BetaHuhn** [comment, marked as answer]
There is no validation because this is only a frontend library. You have to (and should) validate the input on the server.

Validating the input directly in this library would give a false sense of security as an attacker can send his own input directly to the server, bypassing this library entirely.

**Reply by ghost** [reply]
agreed
